### PR TITLE
Keithley6500 trigger_timer validator to narrow

### PR DESCRIPTION
**Pull request by qSaevar**
Changes proposed in this pull request:
- Keithley6500 trigger_timer validator to should allow 0

@astafan8

### CODE REVIEW

File: qcodes/instrument_drivers/tektronix/Keithley_6500.py [hunk 1]
```diff
@@ -171,7 +171,7 @@ def __init__(self, name, address, reset_device=False, **kwargs):
                            get_parser=float,
                            get_cmd='ROUT:SCAN:INT?',
                            set_cmd='ROUT:SCAN:INT {}',
-                           unit='s', vals=Numbers(min_value=0.001, max_value=999999.999))
```
Comment (astafan8) [inline qcodes/instrument_drivers/tektronix/Keithley_6500.py]: and what does the value of 0 mean in this case? trigger immediately? if so, i guess this is to be set using trigger_source parameter, no?

Comment (qSaevar) [reply qcodes/instrument_drivers/tektronix/Keithley_6500.py]: trigger immediately I think. The problem here is that the default value from the device is 0, so if you read the value from the device and try to reapply it, it fails.

Comment (astafan8) [reply qcodes/instrument_drivers/tektronix/Keithley_6500.py]: oh.... ok, could you add that clarification as a comment next to that parameter definition? and then we'll merge.

Comment (qSaevar) [reply qcodes/instrument_drivers/tektronix/Keithley_6500.py]: done

File: qcodes/instrument_drivers/tektronix/Keithley_6500.py [hunk 2]
```diff
@@ -167,11 +167,12 @@ def __init__(self, name, address, reset_device=False, **kwargs):
                                             'bus': 'COMM',
                                             'external': 'EXT'})
 
+        # Control interval between scans, default is 0.
```
Comment (astafan8) [inline qcodes/instrument_drivers/tektronix/Keithley_6500.py]: I meant to add this part :) :
```suggestion
        # Control interval between scans; the default value from the instrument is 0,
        # hence 0 is included in the validator's range of this parameter.
```

**Pull request merged**